public/css/list/schedule-list.less

```diff
@@ -3,6 +3,9 @@
   margin-left: 12em;
   width: ~"calc(100% - 12em)";
 
+  padding-right: 2em; // align the header with the list items
+  padding-left: 0.5em;
```

Comment:
Please adjust the margin instead of adding padding!

The left margin should be 13em in the end and the right margin 1em.

Why only 1em? Because that's a good example of bad CSS. The `.time-grid` of a schedule list item also gets a margin right of 1em even though there's no time scale at the bottom, which was the original reason for this margin. If this margin is only applied in that specific case, it doesn't need to be negated here.